packages/modular-scripts/src/test/utils.ts

```diff
@@ -132,3 +132,34 @@ export function mockInstallTemplate(
     path.join(tempRepoNodeModules, templateName),
   );
 }
+
+/**
+ * Run the Modular cli present at the root path `modularFolder` with a working directory of `cwd`
+ * passing an array of `modularArguments`
+ *
+ * Useful to run modular from source in a different directory (usually a temp directory created for tests)
+ *
+ * @param  modularFolder The root folder in which the modular repo resides
```

Comment:
I feel like the target/cwd could also be a modular repo so I'd make this clearer: 
`@param modularToRun The root modular repo folder containing the modular-scripts to run`